Update for new version

btpence opened the pull request:
Updated a new version yesterday and found this site that hadn't been updated in a while...  Hope this helps.

btpence committed:
Update AbsoluteTelnet version and protocols

btpence committed:
Add links for AbsoluteTelnet SSH client and algorithms

norrisjeremy commented:
Hi @btpence,

Can you confirm if AbsoluteTelnet supports `kex-strict-c-v00@openssh.com` & `kex-strict-s-v00@openssh.com`?
I see mentions in their [Version History](https://www.celestialsoftware.net/version-history/) that indicate it is likely supported (in their 12.11 release).

Thanks!
Jeremy

btpence commented:
Yes it does. And you're correct, that was added in version 12.11 in January of 2024.

Anything else I can answer?

btpence commented:
And looking through other entries, should probably have included:

Protocol extensions: server-sig-algs

Though that's not as clear in the version history

norrisjeremy commented:
> And looking through other entries, should probably have included:
> 
> Protocol extensions: server-sig-algs
> 
> Though that's not as clear in the version history

Can you update the diff to add this, along with adding `ext-info-c` in the `kex` section then?

norrisjeremy commented:
> Yes it does. And you're correct, that was added in version 12.11 in January of 2024.
> 
> Anything else I can answer?

Can you update this diff to include that in the `kex` section?

btpence commented:
Sure

btpence committed:
Update AbsoluteTelnet.md with new options

Added strict kex and key exchange and extension options to the AbsoluteTelnet documentation.

btpence commented:
Done

btpence commented:
And a question if you don't mind..... Do you plan on adding more top-level comparison tables?  It would nice to be able to do client by client comparisons without having to open each clients data sheet.

norrisjeremy commented:
> And a question if you don't mind..... Do you plan on adding more top-level comparison tables? It would nice to be able to do client by client comparisons without having to open each clients data sheet.

I currently don't have any plans: I simply try to keep on top of updates for all the open source SSH implementations as they evolve and update their data here, along with also tracking algorithms proposed for standardization via IETF.

norrisjeremy commented:
> > And a question if you don't mind..... Do you plan on adding more top-level comparison tables? It would nice to be able to do client by client comparisons without having to open each clients data sheet.
> 
> I currently don't have any plans: I simply try to keep on top of updates for all the open source SSH implementations as they evolve and update their data here, along with also tracking algorithms proposed for standardization via IETF.

But you are welcome to submit PRs to implement new tables/pages that you believe would be helpful, and we can try to review them.
